scripts/userstyles.yml

```diff
@@ -368,6 +368,14 @@ userstyles:
     readme:
       app-link: "https://app.graphite.dev"
     current-maintainers: [*isabelroses]
+  gemini:
+    name: Gemini
```

Comment:
```suggestion
  google-gemini:
    name: Google Gemini
```
This is how it is called on Simple Icons and seems accurate since it is a subdomain of google.com and fits in with our other Google userstyles (well just Google Drive lol). The name should be updated in the userstyle metadata as well.